grammar.js

```diff
@@ -930,8 +924,15 @@ module.exports = grammar({
       choice($.value_identifier, $.list_pattern, $.array_pattern),
     ),
 
-    module_unpack_pattern: $ => seq(
-      'module', '(', $.module_identifier, ')',
```

Comment:
`module_unpack_pattern` is not needed. `module_pack` is already listed in `$._pattern` rule